.travis.yml

```diff
@@ -26,6 +29,9 @@ script:
 - tar -xvf travisci/resources/hwc-install.tar.xz -C $HWC_BUILD_DIR
 - mkdir -p $WLD/share/aclocal
 
+# Run cppcheck on codebase
+- cppcheck --quiet --force --error-exitcode=1 .
```

Comment:
Maybe add this to travisci/check_patch.sh? 
We could later add this to the post commit hook to ensure that this is checked while committing.